.github/workflows/uno-updater.yml

```diff
@@ -2,11 +2,10 @@ name: Update Uno Sdk
 
 on:
   schedule:
-    - cron: '0 * * * *'
+    - cron: '0 */6 * * *'
   pull_request:
     branches:
       - main
-      - release/stable/5.3
```

Comment:
I think we should still be executing it for release branches, with `release/stable/*`.

Reply:
Just because we have a backport to a release branch doesn't mean the SDK updater needs to get triggered.